analysis/topEFT/datacard_maker.py

```diff
@@ -194,7 +194,9 @@ def export1d(h, name, cat, fout):
                 for syst,histo in h.items():
                     if syst == 'nominal':
                         fout[name+cat] = hist.export1d(histo)
-                    elif self.do_nuisance and name not in self.syst_special and 'flips' not in name:
+                    elif self.do_nuisance and name not in self.syst_special:
+                        if 'fakes' in name and 'FF' not in syst: continue # Only processes fake factor systs for flips
+                        if 'fakes' not in name and 'FF' in syst: continue # Don't processes fake factor systs for others
```

Comment:
In the pkl file there should not be FF systematics for other samples, but I guess this is a good double check of that.

Reply:
Ok, so the second `if` is probably redundant? Like you said, it's probably good to double check. I don't think it will cause a bottleneck.

Reply:
Right, it's probably redundant, but I guess it does not hurt to keep it (maybe we'd even want to have a warning or something if the second if is true, just to alert us that something is wrong with the pkl file). But I'm completely fine with leaving it as is for now.